receiver/apachereceiver/metadata.yaml

```
@@ -76,6 +88,60 @@ metrics:
       monotonic: true
       aggregation: cumulative
     attributes: [ server_name ]
+  apache.cpu.time:
+    enabled: true
+    description: Jiffs used by processes of given category.
+    unit: "{jiff}"
+    sum:
+      value_type: double
+      input_type: string
+      monotonic: true
+      aggregation: cumulative
+    attributes: [ server_name, cpu_level, cpu_mode ]
+  apache.cpu_load:
```

```suggestion
  apache.cpu.load:
```

Since we've established `apache.cpu` as a "namespece", let's reuse it here.